GWMiniGame/GWMiniGame.cpp

```diff
@@ -0,0 +1,287 @@
+#define _CRT_SECURE_NO_WARNINGS
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+
+char* wordInput() {
+	const int maxWordLength = 5;
+	char buffer[maxWordLength + 1];
+	std::cout << "ENTER : ";
+	std::cin >> buffer;
+	std::cout << std::endl;
+
+	char* result = new char[maxWordLength + 1];
+	for (int i = 0; i <= maxWordLength; ++i) {
+		result[i] = buffer[i];
+	}
+	return result;
+}
+
+int mainMenu(){
+	int value = 0;
+	std::cout << "Greetings in Guess Word Game!" << std::endl;
+	std::cout << "1 - Random Word" << std::endl;
+	std::cout << "2 - Word of the day" << std::endl;
+	std::cout << "0 - Exit" << std::endl;
+	std::cout << "Enter: ";
+	std::cin >> value;
+	std::cout << std::endl;
+	return value;
+}
+
+std::string randomWordFromBD(const char* fileName) {
+	std::fstream file;
+	file.open(fileName, std::ios::in);
+
+	std::string randomWord;
+
+	if (!file) {
+		std::cout << "File not found!\n";
+	}
+	else {
+		std::vector<std::string> words;
+		std::string word;
+
+		while (file >> word) {
+			words.push_back(word);
+		}
+
+		if (words.empty()) {
+			std::cout << "No words found in the file!\n";
+		}
+		else {
+			std::srand(static_cast<unsigned>(std::time(0)));
+			int randomIndex = std::rand() % words.size();
+			randomWord = words[randomIndex];
+		}
+	}
+	file.close();
+	return randomWord;
+}
+
+int getDay(const std::tm * tmObj)
+{
+	return tmObj->tm_mday;
+}
+
+int getMonth(const std::tm * tmObj)
+{
+	return tmObj->tm_mon + 1;
+}
+
+int getYear(const std::tm * tmObj)
+{
+	return tmObj->tm_year + 1900;
+}
+
+std::string getDate() {
+	std::string date;
+	std::string dash = "-";
+	std::time_t t = std::time(nullptr);
+	std::tm* now = std::localtime(&t);
+	date = std::to_string(getDay(now)) + dash + std::to_string(getMonth(now)) + dash + std::to_string(getYear(now));
+	return date;
+}
+
+void writeDate(const char* fileName) {
+	std::fstream file;
+	std::string date = getDate();
+	file.open(fileName, std::ios::out);
+	file << date;
+}
+
+bool checkDate(const char* fileName) {
+	std::fstream file;
+	file.open(fileName, std::ios::in);
+
+	std::string date;
+
+	if (!file) {
+		file.open(fileName, std::ios::out);
+		file.close();
+		return true;
+	}
+	else {
+		std::vector<std::string> dates;
+		std::string d;
+
+		while (file >> d) {
+			dates.push_back(d);
```

Comment:
why would you save all dates there? You just need the most recent date for your case